frontend/src/lib/api/destructionLists.ts

```diff
@@ -74,10 +77,16 @@ export async function createDestructionList(
     )
     .filter((v) => v);
 
+  const allZakenSelected = await getAllZakenSelected(
+    DESTRUCTION_LIST_CREATE_KEY,
```

Comment:
I think this should be a a param as I don't wat to confuse the api with the zaak selection lib, but as I don't know who introduced this, and this is going for so long now. No deal breaker for me right now.

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/310